flytekit/sdk/tasks.py

```diff
@@ -35,7 +36,7 @@ def my_task(wf_params, in1, in2, out1, out2):
     """
 
     def apply_inputs_wrapper(task):
-        if not isinstance(task, _sdk_runnable_tasks.SdkRunnableTask):
+        if not isinstance(task, _sdk_runnable_tasks.SdkRunnableTask) and not isinstance(task, presto_task.SdkPrestoTask):
```

Comment:
add a comment here?  Presto tasks are not sdkrunnable, they're just tasks, but we still want to be able to apply inputs.  If there's ever more, this checking logic should probably be moved to a separate abstract base class or something.